fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/visitor/RelationVisitor.java

```diff
@@ -154,6 +160,10 @@ default R visitPhysicalFileScan(PhysicalFileScan fileScan, C context) {
         return visitPhysicalCatalogRelation(fileScan, context);
     }
 
+    default R visitPhysicalHudiScan(PhysicalHudiScan fileScan, C context) {
+        return visitPhysicalCatalogRelation(fileScan, context);
```

Comment:
if it extends LogicalFileScan, it should
`return visitPhysicalFileScan(fileScan, context);`